contrib/ansible/roles/etcd/tasks/main.yml

```diff
@@ -8,6 +8,11 @@
     - restart etcd
   when: not is_atomic
 
+- name: Clean etcd data directory
+  shell: rm -rf {{ etcd_data_dir }}/*
+  notify:
+    - restart etcd
```

Comment:
That is to say that if the etcd infrastucture changes you need to plow up the data? or in what scope do we need to plow it up? Can we just erase some of the etcd info and leave the Kubernetes data alone? 

Can this be a handler to react to some conditions? Maybe we can pause and prompt that this will happen? http://docs.ansible.com/ansible/pause_module.html